cypress-tests/cypress/e2e/happyPath/platform/eeTestcases/multi-env/multiEnv.cy.js

```diff
@@ -1,662 +1,124 @@
-      .verifyVisibleElement("have.text", "v1")
-      .click();
-    cy.get(multiEnvSelector.currentVersion).verifyVisibleElement(
-      "have.text",
-      "v1"
-    );
-    cy.get(".col-10 > .app-version-name").verifyVisibleElement(
-      "have.text",
-      "v1"
-    );
-    cy.get(multiEnvSelector.createNewVersionButton).verifyVisibleElement(
-      "have.text",
-      "Create new version"
-    );
-
-    verifyTooltip(
-      multiEnvSelector.createNewVersionButton,
-      "New versions can only be created in development"
-    );
+  it("should verify constants visibility and postgresql datasource across all environments in preview", () => {
     cy.forceClickOnCanvas();
-    cy.get(".datasource-picker").should("have.class", "disabled");
-    cy.get(commonEeSelectors.AddQueryButton).should("be.disabled");
-    cy.get(".components-container").should("have.class", "disabled");
-
-    verifyPromoteModalUI("v1", "Staging", "Production");
-    cy.get(commonSelectors.closeButton).click();
-    cy.get(multiEnvSelector.currentEnvName).verifyVisibleElement(
-      "have.text",
-      "Staging"
-    );
-
-    cy.get(commonEeSelectors.promoteButton).click();
-    cy.get(commonSelectors.cancelButton).click();
-    cy.get(multiEnvSelector.currentEnvName).verifyVisibleElement(
-      "have.text",
-      "Staging"
-    );
-
-    cy.get(commonEeSelectors.promoteButton).click();
-    cy.get(commonEeSelectors.promoteButton).eq(1).click();
-    cy.waitForAppLoad();
-    cy.wait(3000);
-
-    cy.get(commonSelectors.warningText).eq(0).verifyVisibleElement(
-      "have.text",
-      "App cannot be edited after promotion. Please create a new version from Development to make any changes."
-    );
-    cy.get(multiEnvSelector.envContainer).should("be.visible");
-    cy.get(multiEnvSelector.currentEnvName)
-      .verifyVisibleElement("have.text", "Production")
-      .click();
-    cy.get(multiEnvSelector.envArrow).should("be.visible");
-    cy.get(multiEnvSelector.currentEnvName).verifyVisibleElement(
-      "have.text",
-      "Production"
-    );
-    cy.get(multiEnvSelector.envNameList)
-      .eq(0)
-      .verifyVisibleElement("have.text", "Development");
-    cy.get(multiEnvSelector.envNameList)
-      .eq(1)
-      .verifyVisibleElement("have.text", "Staging");
-    cy.get(multiEnvSelector.envNameList)
-      .eq(2)
-      .verifyVisibleElement("have.text", "Production");
-
-    cy.get(multiEnvSelector.appVersionLabel).should("be.visible");
-    cy.get('[data-cy="v1-current-version-text"]')
-      .verifyVisibleElement("have.text", "v1")
-      .click();
-    cy.get(multiEnvSelector.currentVersion).verifyVisibleElement(
-      "have.text",
-      "v1"
-    );
-    cy.get(".col-10 > .app-version-name").verifyVisibleElement(
-      "have.text",
-      "v1"
-    );
-    cy.get(multiEnvSelector.createNewVersionButton).verifyVisibleElement(
-      "have.text",
-      "Create new version"
-    );
+    cy.dragAndDropWidget("Button", 400, 400);
```

Comment:
On preview, please just get it promoted using API and open the preview directly.
we don't need to open editor.